com.mojang.Minecraft.json

```diff
@@ -69,13 +45,13 @@
                 {
                     "type": "extra-data",
                     "filename": "Minecraft.tar.gz",
-                    "url": "https://launcher.mojang.com/download/linux/x86_64/minecraft-launcher_2.2.1867.tar.gz",
-                    "sha256": "5e19e3459015fb28bc54df588b6613320281d953bc2f394a28a95465637eeabe",
-                    "size": 76767905,
+                    "url": "https://launcher.mojang.com/download/linux/x86_64/minecraft-launcher_928.tar.gz",
+                    "sha256": "2d7b7ea3835536b6900d0201957c6d96d646354fd071badab2ff750344eba5c6",
+                    "size": 547096,
                     "x-checker-data": {
-                        "type": "html",
+                        "type": "json",
                         "url": "https://launchermeta.mojang.com/v1/products/launcher/6f083b80d5e6fabbc4236f81d0d8f8a350c665a9/linux.json",
-                        "version-pattern": "\"launcher-core\": \\[.*?\"name\": \"(.*?)\".*?\\]",
+                        "version-pattern": ".\"launcher-bootstrap\"[] | {version} | .[] | .name",
                         "url-template": "https://launcher.mojang.com/download/linux/x86_64/minecraft-launcher_$version.tar.gz"
                     }
```

Comment:
This is clearly invalid. JSON checker metadata doesn't support `version-pattern` and `url-template` parameters, it should instead have `version-query` and `url-query` properties for retrieving version and url respectively. Both have to be valid JQ queries (the current `url-template` is not).
Consider using `org.flathub.flatpak-external-data-checker` for testing `x-checker-data`.

Reply:
Oops my bad. I obviously haven't read the docs properly. I'll correct it asap.

Reply:
Fixed it in latest commit